app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -278,12 +278,7 @@ class AttendeeFragment : Fragment(), ComplexBackPressFragment {
             setupPaymentOptions(currentEvent)
             loadEventDetailsUI(currentEvent)
         }
-
-        rootView.register.text =
-            if (rootView.stripeRadioButton.isSelected || rootView.paypalRadioButton.isSelected)
-                getString(R.string.pay_now)
-            else
-                getString(R.string.register)
+        rootView.register.text = if (safeArgs.amount > 0) getString(R.string.pay_now) else getString(R.string.register)
```

Comment:
Why? How can you pay now if the user has selected an offline payment option??

Reply:
PAY NOW should display when the user is paying through an online payment mode. Otherwise, the user is only registering for the event.

Reply:
At first they haven't selected any payment option so I just kept it that way. When they select a payment method, register will be displayed instead of pay now

Reply:
Oh, sorry, get it :+1: